### PR TITLE
Register cdn.zerow.is-a.dev

### PR DESCRIPTION
**Pull request by DevZerow**
Added `cdn.zerow.is-a.dev` using the [CLI](https://www.npmjs.com/package/@is-a-dev/cli).

### CODE REVIEW

**Review by WilliamDavidHarrison — APPROVED** [review]

**Comment by WilliamDavidHarrison** [comment]
Please note: I removed the www.zerow.json file since we do not allow domains starting with `www`.

File: domains/_spf.google.com.zerow.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "DevZerow",
+        "email": "zerow.pro@protonmail.com"
+    },
+
+    "record": {
+        "TXT": "v=spf1 include:_spf.google.com ~all"
```
Comment (WilliamDavidHarrison) [inline domains/_spf.google.com.zerow.json:8]: The file name of this file is invalid.

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

File: domains/default._domainkey.zerow.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "DevZerow",
+        "email": "zerow.pro@protonmail.com"
+    },
+
+    "record": {
+        "TXT": "v=dkim1; p=migfma0gcsqgsib3dqebaquaa4gnadcbiqkbgqdcjg/af7z9z137q+wammo7swvqsegrqtewzqbv9xmtr80i18edp4jpb37ljxlrv2hkfinmqvbs5l8smi80lmz2flm78nmmgw/vnjqytsepjocoisafpanqbqgmtr1kjtexzjtp+kejokhcgan5xabsiatq/m0ybdlvwzl3ztx+qwidaqab;"
```
Comment (WilliamDavidHarrison) [inline domains/default._domainkey.zerow.json:8]: We don't support `._domainkey` subdomains.

File: domains/57fvjjcng24i.zerow.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "DevZerow",
+        "email": "zerow.pro@protonmail.com"
+    },
+
+    "record": {
+        "CNAME": "gv-zsnibvk6cnv5er.dv.googlehosted.com"
```
Comment (WilliamDavidHarrison) [inline domains/57fvjjcng24i.zerow.json:8]: We don't support domain verification.

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Comment by DevZerow** [comment]
Good now ?

**Review by WilliamDavidHarrison — APPROVED** [review]
I've removed `_dmarc` and `_domainkey` since we don't support them.

**Pull request merged**